data_linter/validators/great_expectations_validator.py

```diff
@@ -217,6 +217,8 @@ def _parse_data_to_pandas(
     ]
 
     pandas_kwargs = table_params.get("pandas-kwargs", {})
+    if not pandas_kwargs.get("keep_default_na", True):
+        pandas_kwargs["na_values"] = pandas_kwargs.get("na_values", [''])
```

Comment:
I'm not sure we would want to do this? 

As a user I would expect pandas_kwargs to default to pandas default if I don't set them?

Reply:
Agree, so I've reverted and changed the test instead